src/scss/components/_progress-indicator.scss

```diff
@@ -84,8 +84,13 @@
 
 .#{prefix('progress-indicator-item')} {
   display: flex;
+  @include margin($grid-margin-1, $properties: padding);
   justify-content: flex-start;
 
+  @include bem.modifier('not-applicable') {
+    display: var(--of-progress-indicator-item-not-applicable-display, flex);
+  }
```

Comment:
This CSS-only fix is an accesibility problem, as there's nothing that's now marking the progress item as visually hidden. Screen readers will still read it out, while it's not visible.

I don't think a design token is the right approach here - rather the progress item should be `return null` if it's configured to be hidden for non-applicable steps OR have an `aria-hidden="true"` attribute for accessibility. Both require the backend to expose some information whether progress items should be displayed or not.

Reply:
> You can also use CSS to set display: none or visibility: hidden to hide an element from screen readers and visually.

- https://www.pluralsight.com/guides/how-to-hide-text-from-screen-readers

Reply:
Ah cool, I didn't realize this! Resolving the conversation